.github/workflows/generate-doc-check.yml

```
@@ -48,5 +48,5 @@ jobs:
         run: |-
           sudo apt-get install graphviz
 
-      - name: Compile docs for all Scala versions
-        run: sbt +compile:doc
+      - name: Compile docs&paradox for all Scala versions
+        run: sbt ";Test / compile ;TestJdk9 / compile ;docs/paradox ;+compile:doc"
```

```suggestion
      - name: Compile testClass&docs for all Scala versions
        run: sbt ";TestJdk9 / compile ; +compile:doc"
```

`TestJdk9 / compile` extended from `Test / compile`, I think we could simplify, I have verified in locally.


```
[info] compiling 1 Scala source to /Users/roiocam/IdeaProject/incubator-pekko/cluster-sharding/target/scala-2.13/test-classes ...
[info] compiling 3 Scala sources and 3 Java sources to /Users/roiocam/IdeaProject/incubator-pekko/docs/target/scala-2.13/test-classes ...
[info] compiling 1 Java source to /Users/roiocam/IdeaProject/incubator-pekko/docs/target/scala-2.13/test-classes ...
[info] compiling 2 Scala sources and 2 Java sources to /Users/roiocam/IdeaProject/incubator-pekko/docs/target/scala-2.13/TestJdk9-classes ...

```

And `docs/paradox` doesn't verify anything for #959, and `validaPullRequest` has covered it.